docs/workflow/requirements/macos-requirements.md

```diff
@@ -26,5 +26,5 @@ To build the runtime repo, you will also need to install the following dependenc
 You can install them separately, or you can alternatively opt to install *[Homebrew](https://brew.sh/)* and use the `Brewfile` provided by the repo, which takes care of everything for you. If you go by this route, once you have *Homebrew* up and running on your machine, run the following command from the root of the repo to download and install all the necessary dependencies at once:
 
 ```bash
-brew bundle --no-lock --file eng/Brewfile
+brew bundle --file eng/Brewfile
```

Comment:
`eng/Brewfile` doesn't exist anymore. We can make it:

```suggestion
eng/common/native/install-dependencies.sh
```

Reply:
It's right here? https://github.com/dotnet/runtime/blob/main/eng/Brewfile

Reply:
Ah, we can remove it. We inlined it in install script, I forgot to delete it.